### DISCUSSION TITLE
Pagination Component

### DISCUSSION THREAD

**Discussion by sayshark75** [chakra-ui/chakra-ui#7864] [Ideas]
### Description

I request for a Pagination Component.
It has attributes to take 
current page, 
total pages,
We can define how much number of buttons shown, 
Can be able to add first page button=> true / false,
Can be able to add last page button=> true / false,

First Prev 1 2 3 4 5 Next Last

### Problem Statement/Justification

Yes, Pagination is the major problem in css, and it need a lot of JavaScript/typescript to make a pagination, if a component can take page number which recieved from a API, total number of pages and the limit, and perform some logic behind and create a pagination. It will be very helpful

### Proposed Solution or API

https://mui.com/material-ui/react-pagination/

### Additional Information

https://images.app.goo.gl/veLFdYpaYumoBPyV6

**Comment by AnisTigrini** [comment]
It would be really good to have that kind of component.

**Comment by bytemtek** [comment]
Is there any update about it ?
